src/Slider.js

```diff
@@ -24,11 +20,6 @@ class Slider extends React.Component {
             <input
                 className={classes}
                 type="range"
-                min={min}
-                max={max}
-                value={value}
-                tabIndex="0"
```

Comment:
I just checked their [slider samples](http://www.getmdl.io/components/index.html#sliders-section) and, if you scroll down to "Example," they do show usage without setting `tabindex`